STYLE_GUIDE.md

```diff
@@ -6,6 +6,12 @@ For the most part, following our code style is very simple, we just use
 [black](https://github.com/psf/black) to format code. See our
 [Contributing Guide](CONTRIBUTING.md) for how to run our formatting scripts.
 
+## Naming of Layers and Models
+
+Capitalize all abbreviations, e.g. LSTM not Lstm, KLDivergence not KlDivergence, GPT2, XLMRoberta, etc.
+
+When a specific abbreviation is very common and is pronounceable (acronym), consider it as a standalone word, e.g. Bert, Deberta, etc. In this case, "Bert" is considered as a common noun and not an abbreviation anymore.
```

Comment:
Can you break up this line at 80 characters?

Reply:
Done